cockroach-admin/src/bin/cockroach-admin.rs

```diff
@@ -0,0 +1,79 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Executable program to run the Omicron CockroachDb admin interface (not to be
+//! confused with CockroachDb's built-in HTTP API)
+
+use anyhow::anyhow;
+use camino::Utf8PathBuf;
+use clap::Parser;
+use omicron_cockroach_admin::CockroachCli;
+use omicron_cockroach_admin::Config;
+use omicron_common::cmd::fatal;
+use omicron_common::cmd::CmdError;
+use std::net::SocketAddr;
+use std::net::SocketAddrV6;
+
+#[derive(Debug, Parser)]
+#[clap(name = "cockroach-admin", about = "Omicron CRDB cluster admin server")]
+enum Args {
+    /// Print the OpenAPI Spec document and exit
+    Openapi,
+
+    /// Start the CRDB admin server
+    Run {
+        /// Path to the `cockroach` CLI
+        #[clap(long, action)]
+        path_to_cockroach_binary: Utf8PathBuf,
+
+        /// Socket address for a running cockroach server instance
+        #[clap(long, action)]
+        cockroach_address: SocketAddrV6,
+
+        /// Address on which this server should bin
```

Comment:
```suggestion
        /// Address on which this server should run
```

It's binning about too I suppose